projects/js-packages/components/components/theme-provider/stories/index.stories.tsx

```diff
@@ -48,6 +49,21 @@ const noop = () => {
 	//
 };
 
+/**
+ * Simple function component to wrap `useCopyToClipboard`.
+ *
+ * @param {object} props - Button properties
+ * @param {string} props.text - Text to copy.
+ * @returns {Button} Button
+ */
+function ClipboardButton( {
```

Comment:
It looks like TS still throws an exception here.

Using the React functional component type seems to fix it. I'm no TS expert, and a more appropriate solution may exist.
```ts
const ClipboardButton: React.FC<React.ComponentProps<typeof Button>> = ({
	text,
	...buttonProps
}) => {
	const ref = useCopyToClipboard(text, noop);
	return <Button {...buttonProps} ref={ref} />;
};
```

Reply:
(I updated the PR with this suggestion since it seems better, and will probably merge it before you get a chance to reply, but I'm still curious about the exception I don't see)